test/unit/asrouter/templates/ExtensionDoorhanger.test.jsx

```diff
@@ -0,0 +1,23 @@
+import schema from "content-src/asrouter/templates/CFR/templates/ExtensionDoorhanger.schema.json";
+
+const DEFAULT_CONTENT = {
+  "heading": {
+    "title": "Recommended Extension",
+    "url": "https://support.mozilla.org"
+  },
+  "addon": {
+    "title": "Addon name",
+    "icon": "base64",
+    "author": "Author name"
+  },
+  "content": {
```

Comment:
Since this entire block is going to part of the `content` subsection of the message, isn't this kind of confusing? i.e.

```js
{
  message_id: "content",
  content: {
    heading: "Recommended Extension"
    content: {
      text: "Foo bar baz"
    }
  }
}
```